spotbugs/src/main/java/edu/umd/cs/findbugs/workflow/UnionResults.java

```diff
@@ -157,4 +147,44 @@ public static void main(String[] argv) throws IOException {
         }
     }
 
+    private static SortedBugCollection iterateArguments(List<String> arguments, int argCount, SortedBugCollection results, HashSet<String> hashes) {
+        for (int i = argCount; i < arguments.size(); i++) {
+            String fileName = arguments.get(i);
+            try {
+                SortedBugCollection more = new SortedBugCollection();
+                if (fileName.endsWith(".txt")) {
```

Comment:
@simonjghammer Since the file is spotbugs-argument-file.txt, would it be better here to look for ending with that in the path rather than just .txt?

Reply:
That would be very limiting. I don't think that is necessary. The idea is not only that you can have a single file, you could combine as you wish.

Reply:
but are you not physically doing this...

tempFile = Files.createTempFile("spotbugs-argument-file", ".txt");

So how would it have another name?

Reply:
or am I reading wrong?

Reply:
ok thanks.  I've merged this.